rpc/jsonrpc/types/doc.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+/*
+Package types implements concrete types working with the dcrwallet JSON-RPC API.
+
+Overview
+
+When communicating via the JSON-RPC protocol, all of the commands need to be
+marshalled to and from the the wire in the appropriate format.  This package
+provides data structures and primitives the are registered with dcrjson to ease
```

Comment:
"the" is a typo or grammatical error.

But I wonder whether this whole file should be trimmed.  It reads like it was made for dcrjson and copied over (might have been, didn't look for sure).  It would be much shorter if instead it only discussed that it implements wallet-specific JSON-RPC types and registers them to dcrjson.

Reply:
I copied some of it over.  I can trim it more, but it mainly just provides that for reference.

Reply:
Updated.